phonebuddies/DatabaseConnector.py

```diff
@@ -111,7 +111,11 @@ def get_all_buddies_map(self):
         buddy_email_map = {}
 
         for buddy in self.get_all_buddies():
-            buddy_email_map[buddy.email] = buddy
+            buddy_email_map[buddy.email.lower()] = buddy  # Converts buddy.email to lower before adding to map
+
+        # this is original, which is case-sensitive
+        # for buddy in self.get_all_buddies():
+        #    buddy_email_map[buddy.email] = buddy
```

Comment:
Would you mind removing this? It's useful to keep around while working, but once I merge, it won't make sense to future contributors.